scripts/generate_ssl_debug_helpers.py

```diff
@@ -88,7 +88,8 @@ def _yield_objects(s, d, p, st, end):
                 if has_instance is False:
                     has_instance = True
                     yield pair_start, start_line
-                yield instance.span()[0], instance
+                if instance:
```

Comment:
Minor: This should be unnecessary, no?
`SignatureAlgorithmDefinition.extract` will return a 0/1 length iterator, and `instance` will never be `False`.